services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -317,10 +301,28 @@ def make(cls, node_uuid: UUID) -> "DynamicSidecarNames":
 
 
 class SchedulerData(CommonServiceDetails, DynamicSidecarServiceLabels):
-    service_name: constr(strip_whitespace=True, min_length=2) = Field(
+    service_name: ServiceName = Field(
         ...,
         description="Name of the current dynamic-sidecar being observed",
     )
+    run_id: UUID = Field(
+        default_factory=uuid4,
+        description=(
+            "Used to discriminate between dynamic-sidecar docker resources "
+            "generated during different runs. Sometimes artifacts remain in the"
+            "system after an error. This helps avoiding collisions."
+            "For now used by anonymous volumes involved in data sharing"
```

Comment:
Again, I'm a bit confused by the description.
When I think about RunID I think: ```Uniquely identify the dynamic sidecar session (a.k.a. 2 subsequent exact same services will have a different run id)``` ist this correct?